src/main/java/sg/ncl/MainController.java

```diff
@@ -4918,13 +5006,13 @@ private TeamQuota extractTeamQuotaInfo(String responseBody) {
         amountUsed = amountUsed.multiply(new BigDecimal(charges));   // usage X charges
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this "BigDecimal" constructor [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2129)